pkg/controller/servicebindingrequest/servicebindingrequest_controller.go

```diff
@@ -157,7 +157,7 @@ outerLoop:
 					evs := &corev1.EnvVarSource{
 						SecretKeyRef: sks,
 					}
-					evn := strings.ToUpper(strings.ReplaceAll(instance.Name, "-", "_")) + "_" + strings.ToUpper(strings.ReplaceAll(key, "-", "_"))
+					evn := strings.ToUpper(strings.Replace(instance.Name, "-", "_", -1)) + "_" + strings.ToUpper(strings.Replace(key, "-", "_", -1))
```

Comment:
This is an unrelated change. Please use Go 1.12

Reply:
Ah ok! That explains it. Thanks.